resources/views/licenses/view.blade.php

```diff
@@ -378,8 +378,53 @@
 
               </div> <!-- end row-striped -->
             </div>
-            </div>
-          </div> <!-- end tab-pane -->
+
+          <div class="col-md-3">
+
+            @can('update', $license)
+              <a href="{{ route('licenses.edit', $license->id) }}" class="btn btn-block btn-primary" style="margin-bottom: 10px;">{{ trans('admin/licenses/general.edit') }}</a>
+              <a href="{{ route('clone/license', $license->id) }}" class="btn btn-block btn-primary" style="margin-bottom: 25px;">{{ trans('admin/licenses/general.clone') }}</a>
+            @endcan
+
+            @can('checkout', $license)
+              @if ($license->availCount()->count() > 0)
+                <a href="{{ route('licenses.checkout', $license->id) }}" class="btn-block btn bg-maroon" style="margin-bottom: 10px;" data-toggle="tooltip" title="{{ trans('general.checkout') }}">
+                  Checkout
+                </a>
+                <a href="#" class="btn-block btn bg-maroon" style="margin-bottom: 10px;" style="margin-bottom: 10px;" data-toggle="modal" data-tooltip="true"  data-target="#checkoutFromAllModal">Checkout Remaining to All Users</a>
+              @else
+                  <a href="{{ route('licenses.checkout', $license->id) }}" class="btn btn-block bg-maroon disabled">
```

Comment:
Do you want to do the same tooltip stuff that you do here?: https://github.com/snipe/snipe-it/pull/12849/files#diff-80dc0cd135d35a13f3e0d06c4c7cebf0cf8a2cd941118bbdd3189351559abde5R411

Reply:
None of the tooltips actually work, so....